class-auth.php

```diff
@@ -187,8 +202,16 @@ public function get_token( WP_REST_Request $request ) {
 				return $user_id;
 			}
 			$user = get_user_by( 'id', $user_id );
-		} else {
-			$user = $this->authenticate_user( $username, $password, $custom_auth );
+
+			if ( ! $user ) {
+				$user = new WP_Error(
+					'jwt_auth_user_not_found',
```

Comment:
We should issue a general invalid auth error in this case because exposing whether a user exists or not would open the plugin to a user enumeration security vulnerability.

Reply:
Maybe the general fallback error at the end is sufficient

Reply:
Cool, done that, there was a similar scenario when validating the auth token as well so I also updated that one to say invalid auth tokn instead.